src/web/docusaurus/docs/api-services/dependency-discovery.md

```diff
@@ -0,0 +1,239 @@
+---
+sidebar_position: 5
+---
+
+# Dependency Discovery Service
+
+## Overview
+
+The dependency discovery service provides information about dependencies of Telescope. That information is mostly related to GitHub issues for the purposes of searching what issues to contribute to open-source projects.
+
+## API endpoints
+
+### Get a list of dependency names
+
+To start with the service, you can get the names of dependencies that Telescope uses.
+Be aware that this route will return all the names that the service currently stores (pagination is not available, as of yet).
+
+```
+GET /projects
+```
+
+#### Parameters
+
+There are no parameters to provide.
+
+#### Code Samples
+
+##### Shell
+
+```bash
+curl http://localhost/v1/dependency-discovery/projects
+```
+
+##### JavaScript
+
+- With `fetch`
+
+```js
+fetch('http://localhost/v1/dependency-discovery/projects');
+```
+
+#### Responses
+
+#### Successful response
+
+```
+Status: 200 OK
+```
+
+It will return a JSON-like array of strings, where each element is a dependency name that the service recognizes.
+
+##### Unsuccessful response
+
+While unexpected, the service could provide a status error code. A few reasons as to why this could happen:
+
+- the service crashed,
+- the service took to long to give a response (which could be because it is stuck on reading a super long file!),
+- among other reasons.
```

Comment:
`or something else.`

Perhaps my version isn't great either, but `among other reasons` is really out of place here